### ISSUE TITLE
validate called in initialize, not consistent with docs

### ISSUE THREAD

**Issue by pmackay** [theodi/csvlint.rb#114]
The README suggests to construct a Validator and then call `validator.validate`. This is not necessary as `validate` is called in `initialize`. This results in double the number of recorded issues.

Suggest either fix the docs or remove call to `validate` in `initialize`?

**Comment by waded** [comment]
This bit me too.

**Comment by glacier** [comment]
yes it would be nice to update the docs. I had to look at the code to figure this out. thanks!
